src/main/resources/db/migration/V17__add_direct_messages_conversation_id.sql

```
@@ -0,0 +1,20 @@
+ALTER TABLE direct_messages ADD COLUMN conversation_id VARCHAR(73);
+
+UPDATE direct_messages
+SET conversation_id = CASE
+                          WHEN sender_id::text < receiver_id::text
+        THEN sender_id::text || '_' || receiver_id::text
+                          ELSE receiver_id::text || '_' || sender_id::text
+    END;
+
+ALTER TABLE direct_messages ALTER COLUMN conversation_id SET NOT NULL;
+
+DROP INDEX IF EXISTS IDX_direct_messages_sender_receiver_created_id;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)